site/en/tutorials/reinforcement_learning/actor_critic.ipynb

```diff
@@ -174,6 +174,7 @@
       "source": [
         "%%bash\n",
         "# Install additional packages for visualization\n",
+        "sudo apt-get update\n",
```

Comment:
In what kind of setup is this required? In a container?

Reply:
If the notebook is designed for running in Colab first, this may not be necessary. I've tested it and it works out of the box. However, it's a valid point. Is there a final decision on this? @lamberta @MarkDaoust

Reply:
We do run this inside a container for testing and publishing.

You can remove the `apt-get update` since we do that in the container. If we hit any errors, we will add the `apt-get update`.

Reply:
@yashk2810: That's the whole PR.

Reply:
Okay, since it runs end-to-end, I am going to close the PR.